src/OrchardCore.Modules/Orchard.Tokens.Content/Services/ISlugService.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Orchard.Tokens.Content.Services
+{
+    public interface ISlugService
```

Comment:
Where should be the new project placed? In the `OrchardCore` directory or in the `OrchardCore.Modules` directory?